### DISCUSSION TITLE
Commendable feature on the beta version of Github.

### DISCUSSION THREAD

**Discussion by Kamsi-yonna** [community/community#102435] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello Github 👋🏿! I just saw the lists feature on the beta version and it's such a useful addition for me because my starred repos were getting quite cumbersome 😅. This sort of helps me to group and easily sort my starred repositories. Thanks and kudos to the team.
In addition to this feature. One feature I would love to commend and maybe see on the list is the drag and drop feature which is on the pinned section on a person's profile. 
I feel this would take the experience a step further. Going through a long list of repos can get overwhelming. Being able to set them them in that grid arrangement and being able to use the drag tool to arrange,sort or group these starred repos would be a nice addition. 
For this, maybe add a toggle button to switch the display mode to grid or list view. 
Thanks for all you do. I hope this gets to you 😬😬😬

**Comment by Malekaljamay** [comment]
بوت لاخطراق

**Comment by dinhtuananhne** [comment]
#1

**Reply by BMWhacker** [reply]
https://www.facebook.com/profile.php?id=100053705798857&mibextid=sCpJLy

**Comment by Gracielaminely** [comment]
> ### Select Topic Area
> 
> Product Feedback
> 
> ### Body
> 
> Hello Github 👋🏿! I just saw the lists feature on the beta version and it's such a useful addition for me because my starred repos were getting quite cumbersome 😅. This sort of helps me to group and easily sort my starred repositories. Thanks and kudos to the team.
> In addition to this feature. One feature I would love to commend and maybe see on the list is the drag and drop feature which is on the pinned section on a person's profile. 
> I feel this would take the experience a step further. Going through a long list of repos can get overwhelming. Being able to set them them in that grid arrangement and being able to use the drag tool to arrange,sort or group these starred repos would be a nice addition. 
> For this, maybe add a toggle button to switch the display mode to grid or list view. 
> Thanks for all you do. I hope this gets to you 😬😬😬

**Comment by RandyMateusz** [comment]
- [ ] **~~@Kamsi-yonna ~~**

**Comment by BMWhacker** [comment]
https://www.facebook.com/profile.php?id=100053705798857&mibextid=sCpJLy

**Comment by BMWhacker** [comment]
https://www.facebook.com/profile.php?id=100053705798857&mibextid=sCpJLy
